src/tag_version/core.py

```
@@ -0,0 +1,160 @@
+    except subprocess.CalledProcessError as e:
+        error_message = e.stderr if e.stderr else str(e)
+        # Don't print here, let the caller handle the error display
+        raise RuntimeError(
+            f"Command '{e.cmd}' returned non-zero exit status {e.returncode}.\n{error_message}"
+        )
+
+
+def push_git_tag(tag: str) -> tuple[bool, str]:
+    """Push git tag to remote repository"""
+    try:
+        result = subprocess.run(
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)